components/tikv_util/src/yatp_pool/mod.rs

```diff
@@ -377,6 +377,17 @@ impl<T: PoolTicker> YatpPoolBuilder<T> {
         )
     }
 
+    pub fn build_multi_level_future_pool(self) -> FuturePool {
+        let name = self
+            .name_prefix
+            .clone()
+            .unwrap_or_else(|| "yatp_pool".to_string());
+        let size = self.core_thread_count;
+        let task = self.max_tasks;
+        let pool = self.build_multi_level_pool();
+        FuturePool::from_pool(pool, &name, size, task)
+    }
+
     pub fn build_multi_level_pool(self) -> ThreadPool<TaskCell> {
```

Comment:
where is it used after this pr, only test case?

Reply:
`build_multi_level_future_pool` also use it.